### PR TITLE
Set up prometheus pushgateway

### PR DESCRIPTION
**Pull request by lbarcziova**

### CODE REVIEW

File: openshift/pushgateway.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+          volumeMounts:
+            - mountPath: /data
+              name: pushgateway-data
+          resources:
+            limits:
+              cpu: 50m
+              memory: 50Mi
+            requests:
+              cpu: 25m
+              memory: 25Mi
```
Comment (jpopelka) [inline openshift/pushgateway.yml.j2:42]: Not sure this will work on Openshift Online, see comment in 746cf271692

You can try, however.

Comment (lbarcziova) [reply openshift/pushgateway.yml.j2:42]: will change it

File: openshift/nginx.yml.j2 [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+      volumes:
+      - name: nginx-config
+        configMap:
+          name: nginx-config
+      - name: metrics-secret
+        secret:
+          secretName: metrics-secret
+      containers:
+      - name: nginx
+        image: nginxinc/nginx-unprivileged
```
Comment (lbarcziova) [inline openshift/nginx.yml.j2:28]: I tried to use [nginx image](https://hub.docker.com/_/nginx) before, but it required super-user privileges, with [this image](https://hub.docker.com/r/nginxinc/nginx-unprivileged) it works fine, do you know any better alternative?

File: openshift/nginx.yml.j2 [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+spec:
+  host: "{{ 'workers.stg.packit.dev' if deployment == 'stg' else 'workers.packit.dev' if deployment == 'prod'}}"
+  port:
+    targetPort: nginx-http
+  to:
+    kind: Service
+    name: nginx
+  tls:
+    insecureEdgeTerminationPolicy: Redirect
+    termination: passthrough
```
Comment (csomh) [inline openshift/nginx.yml.j2:92]: In order for this to work nginx will need to be configured to use the wildcard cert used by packit-service.

Comment (lbarcziova) [reply openshift/nginx.yml.j2:99]: done

**Review by csomh — APPROVED** [review]
LGTM, thank you!

**Pull request merged**